Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -710,6 +710,11 @@ open class LottieAnimationView: LottieAnimationViewBase {
     animationLayer?.getOriginalValue(for: keypath, atFrame: atFrame)
   }
 
+  /// Gets all child keypaths.
+  public func getHierarchyKeypaths() -> [String] {
```

Comment:
An API like this would be more idiomatic:

```suggestion
  public var allAnimationKeypaths: [String] {
```